pkg/converter/convert_unix.go

```diff
@@ -858,6 +877,22 @@ func convertIndex(ctx context.Context, cs content.Store, orgDesc ocispec.Descrip
 	return newIndexDesc, nil
 }
 
+// isNydusImage checks if the last layer is nydus bootstrap,
+// so that we can ensure it is a nydus image.
+func isNydusImage(manifest *ocispec.Manifest) bool {
+	layers := manifest.Layers
+	if len(layers) != 0 {
+		desc := &layers[len(layers)-1]
+		if desc.Annotations == nil {
+			return false
+		}
+		if _, ok := desc.Annotations[LayerAnnotationNydusBootstrap]; ok {
+			return true
+		}
```

Comment:
It seems that we can replace this with the `IsNydusBootstrap` method?

Reply:
Thanks! Fixed.